tests/bench_sizeof_coretypes/main.c

```diff
@@ -105,6 +105,7 @@ int main(void)
 #endif
 #if defined(DEVELHELP) || defined(SCHED_TEST_STACK) || defined(MODULE_MPU_STACK_GUARD)
     P(stack_start);
+    P(stack_size);
```

Comment:
This is only available with `defined(DEVELHELP)`. This would lead to a compilation error with `SCHED_TEST_STACK` or `MODULE_MPU_STACK_GUARD` defined, but with `DEVELHELP` undefined:

```
$ CFLAGS=-DSCHED_TEST_STACK DEVELHELP=0 make -C tests/bench_sizeof_coretypes/
make: Entering directory '/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes'
Building application "tests_bench_sizeof_coretypes" for "native" with MCU "native".

/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c: In function ‘main’:
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:7: error: ‘thread_t’ {aka ‘struct _thread’} has no member named ‘stack_size’; did you mean ‘stack_start’?
     P(stack_size);
       ^~~~~~~~~~
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:43:59: note: in definition of macro ‘P’
                        (unsigned)sizeof(((thread_t *) 0)->NAME), \
                                                           ^~~~
In file included from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/kernel_defines.h:24,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/panic.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/assert.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/cib.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:25:
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:7: error: ‘thread_t’ {aka ‘struct _thread’} has no member named ‘stack_size’; did you mean ‘stack_start’?
     P(stack_size);
       ^~~~~~~~~~
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:5: note: in expansion of macro ‘P’
     P(stack_size);
     ^
make[1]: *** [/home/mlenders/Repositories/RIOT-OS/RIOT/Makefile.base:83: /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/bin/native/application_tests_bench_sizeof_coretypes/main.o] Error 1
make: *** [/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/../../Makefile.include:434: /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/bin/native/application_tests_bench_sizeof_coretypes.a] Error 2
make: Leaving directory '/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes'

$ USEMODULE=mpu_stack_guard DEVELHELP=0 make -C tests/bench_sizeof_coretypes/
make: Entering directory '/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes'
Building application "tests_bench_sizeof_coretypes" for "native" with MCU "native".

/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c: In function ‘main’:
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:7: error: ‘thread_t’ {aka ‘struct _thread’} has no member named ‘stack_size’; did you mean ‘stack_start’?
     P(stack_size);
       ^~~~~~~~~~
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:43:59: note: in definition of macro ‘P’
                        (unsigned)sizeof(((thread_t *) 0)->NAME), \
                                                           ^~~~
In file included from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/kernel_defines.h:24,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/panic.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/assert.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/core/include/cib.h:25,
                 from /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:25:
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:7: error: ‘thread_t’ {aka ‘struct _thread’} has no member named ‘stack_size’; did you mean ‘stack_start’?
     P(stack_size);
       ^~~~~~~~~~
/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/main.c:108:5: note: in expansion of macro ‘P’
     P(stack_size);
     ^
make[1]: *** [/home/mlenders/Repositories/RIOT-OS/RIOT/Makefile.base:83: /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/bin/native/application_tests_bench_sizeof_coretypes/main.o] Error 1
make: *** [/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/../../Makefile.include:434: /home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes/bin/native/application_tests_bench_sizeof_coretypes.a] Error 2
make: Leaving directory '/home/mlenders/Repositories/RIOT-OS/RIOT/tests/bench_sizeof_coretypes'
```

Reply:
right, good catch, thx!